aries-site/src/pages/templates/pagination.mdx

```diff
@@ -0,0 +1,76 @@
+import { Example } from '../../layouts';
+import {
+  PaginationCardsExample,
+  PaginationListExample,
+  PaginationTableExample,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/templates/pagination/PaginationTableExample.js"
+  docs="https://v2.grommet.io/pagination?theme=hpe#props"
+  figma="https://www.figma.com/file/slRqNvhJub0QlgN46zSXoY/HPE-Pagination-Component?node-id=0%3A1"
+>
+  <PaginationTableExample />
+</Example>
+
+## When to use
+
+Use pagination when a user’s goal is to find a specific item rather than to peruse the full set of data. By breaking the data up into smaller, consistently sized groups, the user is more easily able to jump to specific sections of the data.
+
+If you expect the user will be scrolling and perusing large amounts of the data set, consider using [Infinite Scroll]().
```

Comment:
think you forgot the link

Reply:
Good catch. We don't have a page yet for this so I'll add link to Grommet docs